blackjax/sgmcmc/gradients.py

```diff
@@ -0,0 +1,55 @@
+from typing import Callable
+
+import jax
+import jax.numpy as jnp
+
+from blackjax.types import PyTree
+
+
+def grad_estimator(
+    logprior_fn: Callable, loglikelihood_fn: Callable, data_size: int
+) -> Callable:
+    """Builds a simple gradient estimator.
+
+    This estimator first appeared in [1]_.
+
+    Parameters
+    ----------
+    logprior_fn
+        The log-probability density function corresponding to the prior distribution.
+    loglikelihood_fn
```

Comment:
Here I should specify the expected signature for `loglikelihood_fn` and `logprior_fn` as it is a potential source of confusion.